tests/test_frame.py

```diff
@@ -998,28 +999,31 @@ def test_types_from_dict() -> None:
 
 
 def test_pipe() -> None:
-    if TYPE_CHECKING:  # skip pytest
+    def foo(df: pd.DataFrame) -> pd.DataFrame:
+        return df
+
+    val = (
+        pd.DataFrame(
+            {
+                "price": [10, 11, 9, 13, 14, 18, 17, 19],
+                "volume": [50, 60, 40, 100, 50, 100, 40, 50],
+            }
+        )
+        .assign(week_starting=pd.date_range("01/01/2018", periods=8, freq="W"))
+        .resample("M", on="week_starting")
+        .pipe(foo)
+    )
 
-        def foo(df: pd.DataFrame) -> pd.DataFrame:
-            return df
+    # Cant' check actual return type because it is DatetimeIndexResampler which is
+    # not part of the public API
+    assert_type(val, Union[Scalar, pd.Series, pd.DataFrame, Resampler])
 
+    if TYPE_CHECKING:  # skip pytest
```

Comment:
This is old code, not mine.

Reply:
It was skipped because of `jinja2` not being installed, so you could make that a dev dependency and remove it (relates to the `Styler` PR you are doing)